src/main/scala/io/hydrosphere/serving/manager/domain/monitoring/MonitoringConfiguration.scala

```diff
@@ -0,0 +1,16 @@
+package io.hydrosphere.serving.manager.domain.monitoring
+
+import spray.json._
+
+case class MonitoringConfiguration(batch_size: Int = 100) {
+
+}
+
+
+object MCProtocol extends DefaultJsonProtocol {
+  implicit val MCFormat = jsonFormat1(MonitoringConfiguration)
+}
+
+object MCDefault {
+  val JsValue = """{ "batch_size": 100 }""".parseJson
```

Comment:
Incorrect initialization here.

We need default value for type MonitoringConfiguration. Create it explicitly, and make as `val` in object companion.

Reply:
Added, could you check a new version?